### PR TITLE
Improved readbility

### PR DESCRIPTION
**Pull request by abhishek213-alb**
Issue no #26

Fixed typos & grammatical errors

### CODE REVIEW

**Comment by abhishek213-alb** [comment]
@BlankCheng Hello sir  I have updated it

File: README.md [hunk 1]
```diff
@@ -48,12 +48,12 @@ We have now implemented three agents in OpenAgents, and we host them on [demo](h
 2. Plugins Agent with 200+ daily tools; 
 3. Web Agent for autonomous web browsing.
 
-OpenAgents can analyze data, call plugins, control your browser as ChatGPT Plus, but with OPEN Code for
+OpenAgents can analyze data, call plugins, and  control your browser as ChatGPT Plus, but with OPEN Code for
```
Comment (koalazf99) [inline README.md:51]: extra space

**Review by koalazf99 — CHANGES_REQUESTED** [review]

**Comment by koalazf99** [comment]
Thank you @abhishek213-alb !!! 

As mentioned in #26 , We will handle these typo-fixing PRs in "batches" regularly to make commit history cleaner.

So you can expect this PR will be finally handled with other typo-fixing PRs together after some time.

**Review by abhishek213-alb — COMMENTED** [review]
Fixed extraspace

**Comment by abhishek213-alb** [comment]
@koalazf99 sir I have fixed extra spacing

**Comment by abhishek213-alb** [comment]
@BlankCheng sir can you review it

**Comment by abhishek213-alb** [comment]
@koalazf99 sir you have closed instead of merging

**Pull request closed without merging**